WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -383,8 +383,10 @@ local function UpdateProgressFromState(self, minMaxConfig, state, progressSource
   local remainingProperty = progressSource[8]
 
   if progressType == "number" then
-    local value = state[property] or 0
-    local total = totalProperty and state[totalProperty] or 0
+    local value = state[property]
+    if type(value) ~= "number" then value = 0 end
+    local total = totalProperty and state[totalProperty]
+    if type(total) ~= "number" then total = 0 end
```

Comment:
one of the rare cases where a comment might be good to explain why we are doing that?